llvm/include/llvm/Transforms/Vectorize/LoopVectorizationLegality.h

```diff
@@ -391,25 +391,22 @@ class LoopVectorizationLegality {
 
   /// Returns true if the loop has an uncountable early exit, i.e. an
   /// uncountable exit that isn't the latch block.
```

Comment:
Might be good to clarify there must be single edge

```suggestion
  /// Returns true if the loop has exactly one uncountable early exit, i.e. an
  /// uncountable exit that isn't the latch block.
```

Reply:
Done